src/Collections/Sites.php

```diff
@@ -190,41 +179,74 @@ public function findUuidByName($name)
     }
 
     /**
+     * Get the site's UUID given either it's site name or it's UUID
+     *
+     * @param $id
+     * @return string
+     */
+    public function findUuid($id)
```

Comment:
Yeah. The whole thing is a bit clunky. `findUuidByName` returns an object not a uuid and is a public function meaning it's part of the API. I think it should be internal-only or do return what it implies it should.

I think there should be 1 (public of private) function that takes either a uuid or a site name as input and returns a uuid. I don't really care what we call it, but we need it to normalize site identifiers whenever they are used.